t/08-autotest.t

```diff
@@ -316,9 +316,10 @@ subtest 'load test successfully when CASEDIR is a relative path' => sub {
 };
 
 subtest python => sub {
-    stderr_like {
+    combined_like {
         lives_ok { autotest::loadtest('tests/pythontest.py') } 'can load test module'
-    } qr{scheduling pythontest tests/pythontest.py}, 'python pythontest module referenced';
+    } qr{(?|scheduling pythontest tests/pythontest|Using python version)}, 'python pythontest module referenced';
```

Comment:
That regex would even match an empty string.
I think what you would want here is:
```suggestion
    } qr{scheduling pythontest tests/pythontest.*Using python version}s, 'python pythontest module referenced';
```